docs/help/git_webhooks.md

```diff
@@ -0,0 +1,57 @@
+## Webhooks
+
+### Give the bot an SSL-certificate
+
+#### Self-signed certificate
+
+***WARNING!*** Your git host may not accept self-signed certificates.
```

Comment:
This seems to be the case at least with https://gitea.blesmrt.net/ which gave me `Delivery: Post https://relpda.mikaela.info:5000/api/gitea?key=: x509: certificate signed by unknown authority`

Reply:
oh, of course I am not sure if this was just that the bot was giving partial chain as I originally didn't give it a `fullchain.pem`